src/views/Pipeline.vue

```diff
@@ -7,7 +7,7 @@
       </ion-toolbar>
 
       <ion-toolbar>
-        <div>
```

Comment:
Remove extra space and improve indetation.

Reply:
ok ma'am

Reply:
Improved indentation and removed extra spaces.